src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+    /**
+     * @return printable string of each object
+     */
```

Comment:
Try to replace `@return ...` lines with proper header comments if header comments can also describe the method as well as the `@return...` lines (readability).